### ISSUE TITLE
Confirming when opening a link

### ISSUE THREAD

**Issue by SaySayTakamura** [bluesky-social/social-app#1529]
**Is your feature request related to a problem? Please describe.**

I think this one falls under both Feature Request and Bug Report.

Yesterday i clicked on someone's handle (username.blue, username is just a placeholder) and out of the blue (all puns not intended) it opens the browser where the site is the user handle, but it does not exists.

I don't know if it lacked the "@" or it was a bug, since i was on someone else's profile

**Describe the solution you'd like**

It would be nice to check if the handle exists before.

If not shows a message for the user.

Also, it would be nice to warn the user when opening a link, something like "This link leads to an external website, are you sure you want to be redirected?"

Sorry for the bothering once again and hope this is useful.

**Comment by pfrazee** [comment]
Yep this is actually a planned feature, exactly because of what you're describing. Appreciate the issue!

**Comment by mozzius** [comment]
I think this is done?
